content/Watsonx/CodeAssistant/101.md

```diff
@@ -0,0 +1,78 @@
+---
+title: '101: Ansible Lightspeed Environment Setup'
+timeToComplete: 15
+updated: 2023-09-13
+---
+
+# 101: Ansible Lightspeed environment setup in VS Code
+
+## Goal
+
+The goal of this lab is to install the Ansible Lightspeed extension and enable and authenticate the extension in VS Code. 
+
+## Install Ansible extension in VS Code
+
+1. Open the VS Code application.
+2. Click the **Extensions** icon on the left sidebar.
+
+    ![](./images/settings-icon.png)
+3. Search for **Ansible** in the search box and then click the **Install** button.
+
+    ![](./images/search-for-ansible.png)
+
+## Enable the Ansible Lightspeed extension
+
+1. After installing the Ansible extension, click on the little gear icon and select the **Extension Setting** option.
+
+    ![](./images/gear-icon.png)
+2. In the Search settings bar, type **lightspeed**
+3. If they are not already selected, check all of the following boxes:
+ - Ansible > Lightspeed: **Enabled**
+ - Ansible > Lightspeed > Suggestions: **Enabled**.
+
+    ![](./images/enable-lightspeed.png)
+
+**Note:** For the purpose of this demo, we will disable Ansible Lint and Ansible Execution Environment.
+
+1. In the Search settings bar, type **ansible lint**. If it is not already unselected, *uncheck* the **Ansible > Validation > Lint: **Enabled**** box.
+
+    ![](./images/disable-ansible-lint.png)
+
+2. In the Search settings bar, type **ansible execution**. If it is not already unselected, *uncheck* the **Ansible > Execution Environment: **Enabled**** box.
+
+    ![](./images/disable-ansible-exec.png)
+
+## Authenticate Ansible Lightspeed with GitHub
+
+1. Open the Ansible extension icon on the left-hand side of the Visual Studio Code interface and click **Connect**.
+
+    ![](./images/connect-ansible.png)
+
+2. Follow the prompts to sign in to Ansible Lightspeed with IBM watsonx Code Assistant using your GitHub credentials.
+
+    a. Click on the **Allow** button.
+
+    ![](./images/ansible-sign-in.png)
+
+    b. Click on the **Open** button.
+
+    ![](./images/external-website.png)
+
+    c. Click on the **Log in with GitHub** button.
+
+    ![](./images/github-login.png)
+
+    d. Click on the **Authorize** button
+
+    ![](./images/authorize-ansible.png)
+
+    e. Click on the **Authorize ansible** button
+
+    ![](./images/authorize-ansible-github.png)
+
```

Comment:
After clicking "Authorize ansible" in the screenshot above, I went to a terms agreement page.  Should we include a step for agreeing to those terms?
url: https://c.ai.ansible.redhat.com/community-terms/?partial_token=10bf91675e894bb8836910791d941de1

<img width="1167" alt="Screen Shot 2023-09-17 at 2 44 46 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/7364498/ed1e2fb4-924f-47a6-b963-4a94b7515c10">
<img width="1140" alt="Screen Shot 2023-09-17 at 2 45 00 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/7364498/af1c7236-d770-44ea-aea7-c1f4270894cb">

Reply:
Ahhh, after clicking the Agree button, then I get the "Authorize Ansible Lightspeed for VS Code" dialog that is show above in step d.  @tkmonson Can you pay close attention to the order you face when you go through the steps?   Not sure if this is unique to me or if we should update the order of steps. 
<img width="586" alt="Screen Shot 2023-09-17 at 2 45 43 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/7364498/eca3b290-482f-45a6-87e1-38fb509a5135">

Reply:
@ridgill I saw things in the same order as you. "Log in with GitHub," then agreeing to Terms of Service, then "Authorize Ansible Lightspeed for VS Code."

Reply:
I added screenshots for the terms and agreement page. I retried the flow again and I wasn't presented with that page. I think I agreed to it before.